server/SaveLoad.cpp

```diff
@@ -490,6 +490,10 @@ void LoadGame(const std::string& filename, ServerSaveGameData& server_save_game_
             }
         }
 
+        for (auto it : empire_manager) {
```

Comment:
don't need the `{}` for a one-line loop body

Reply:
I am used to using them always, but I can adapt...